garrysmod/lua/includes/modules/draw.lua

```diff
@@ -132,15 +134,15 @@ function MultiColorText( font, x, y, xAlign, yAlign, ... )
 	local w, h = 0, 0
 
 	if ( xAlign != TEXT_ALIGN_LEFT ) then
-		local fullText = ""
-		
+		table.Empty( MultiColorText_StringTable )

```

Comment:
table.Empty uses pairs

Reply:
Just create a new table every time.

Reply:
Creating a new table would be faster. Clearing the table is stuitable for large table